web/src/components/index.css

```diff
@@ -8,6 +8,7 @@
 @import url('listen-box/listen-box.css');
 @import url('modal/modal.css');
 @import url('pages/data/data.css');
+@import url('pages/kiosk/kiosk.css');
```

Comment:
We transitioned to requiring css files in their related components. The files that are still required in `index.css` are the older pages (where I wasn't sure how it'd affect the cascade, as it changes the order).